### PR TITLE
CEPH-83573650- Verify parent snap deletion after clone flattening

### PR DESCRIPTION
**Pull request by rkhambad**
Ticket : https://issues.redhat.com/browse/RHCEPHQE-8477
This PR includes Automation of Parent snap deletion after flattening the clone.

Polarion Link : https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573650

File changed :
[suites/pacific/rbd/tier-2_rbd_regression.yaml] - Added test suite
[suites/quincy/rbd/tier-2_rbd_regression.yaml] - Added test suite

New module file added :
tests/rbd/rbd_clone_delete_parent_snapshot.py

success log : 
5.3 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1A6VQY 
6.0 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-IB2YLP/

### CODE REVIEW

**Review by sunilangadi2 — DISMISSED** [review]
LGTM.

File: tests/rbd/rbd_clone_delete_parent_snapshot.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    flatten the clone, unprotect the parent snap, delete the parent snap.
+
+    Test Case Flow -
+    1. Create a Rep pool and an Image
+    2. Create Snapshot and protect, and clones from it.
+    3. Create a snapshot of clone.
+    4. Flatten the clone.
+    5. Unprotect and delete the parent snapshot
+    6. Repeat above steps on EC pool
+    """
```
Comment (manasagowri) [inline tests/rbd/rbd_clone_delete_parent_snapshot.py:18]: Could you please correct the indentation on this line

Comment (rkhambad) [reply tests/rbd/rbd_clone_delete_parent_snapshot.py]: Done.

**Review by manasagowri — APPROVED** [review]
LGTM.

**Pull request merged**